pipeline-templates/job/code-build.yml

```diff
@@ -23,25 +24,25 @@ jobs:
     - template: azure-pipelines-templates/build/step/dependency-check.yml@das-platform-building-blocks
 
     - task: DotNetCoreCLI@2
-      displayName: Publish - dotnet publish ${{ SolutionBaseName }}
+      displayName: "Publish $(SolutionBaseName)"
       inputs:
         command: publish
         publishWebProjects: false #set to false as this setting (which defaults to true) will parse the entire repo for web projects
-        projects: src/${{ SolutionBaseName }}/${{ SolutionBaseName }}.csproj
-        arguments: -o $(build.artifactstagingdirectory)/publish -c ${{ BuildConfiguration }} --no-build
+        projects: src/$(SolutionBaseName)/$(SolutionBaseName).csproj
+        arguments: "-o $(build.artifactstagingdirectory)/publish -c $(BuildConfiguration) --no-build"

```

Comment:
das-github-template has BuildConfiguration passed as a parameter from azure-pipelines.yml for explicitness

Reply:
@jack-cloud-platform so do i need to remove it ?

Reply:
das-github-template standards should ideally not be moved away from. You can propose changes to das-github-template though

Reply:
@jack-cloud-platform No worries, i will check it out tomorrow. Thanks for the review